Nuvoton-Israel/zephyr
# I3c controller driver

1. Add I3c controller driver support
2. Add test application, tests\boards\npcm400f_evb for I3c controller API
3. merge a patch for stack warning in arch\arm\include\aarch32\cortex_m\stack.h

To test i3c,
1. You should comment PECI and uncomment for i3c default pin control in \boards\arm\npcm400f_evb\npcm400f_evb_defconfig #CONFIG_PECI=y #CONFIG_PECI_NPCM4XX=y

   CONFIG_I3C=y CONFIG_I3C_NPCM4XX=y

2. You must disable i2c6a node in \boards\arm\npcm400f_evb\npcm400f_evb.dts to prevent interrupt sharing. &i2c6a { clock-frequency = <200000>; status = "disabled"; };

3. connect LSM6DSO to I3C1
